.rubocop.yml

```diff
@@ -79,3 +79,17 @@ Style/ParenthesesAroundCondition:
 
 Style/PreferredHashMethods:
   Enabled: false
+
+Minitest/AssertEmptyLiteral:
+  Enabled: true
+
+Minitest/AssertNil:
+  Exclude:
+    - 'test/graphql/queries/personal_access_token_query_test.rb'
```

Comment:
Why is this excluded? In the TODO it was excluded to silence the error, but I'd rather fix the errors it would raise.